frontend/src/scenes/insights/trendsLogic.ts

```diff
@@ -280,6 +282,35 @@ export const trendsLogic = kea<trendsLogicType<FilterType, ActionType, TrendPeop
         setDisplay: async ({ display }) => {
             actions.setFilters({ display })
         },
+        refreshCohort: () => {
+            cohortLogic({
+                cohort: {
+                    id: 'new',
+                    groups: [],
+                },
+            }).actions.setCohort({
+                id: 'new',
+                groups: [],
+            })
+        },
+        saveCohortWithFilters: ({ cohortName }) => {
+            if (values.people) {
+                const { label, action, day, breakdown_value } = values.people
+                const filterParams = parsePeopleParams({ label, action, day, breakdown_value }, values.filters)
+                const cohortParams = {
+                    is_static: true,
+                    name: cohortName,
+                }
+                cohortLogic({
```

Comment:
This feels slightly sketchy.

Reply:
yup. I originally had this in the `connect` handler, but I think kea typegen doesn't handle logics that are initialized with props in connect yet so it was erroring unless I did this. Need to QA to make sure the logics work as expected here